website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Firebolt configurations"
+id: "firebolt-configs"
+---
+
+## Using dbt-firebolt
```

Comment:
@ericf-firebolt This is an example of a header that is no longer relevant due to the change in context between the dbt docs vs our readme - please remove